src/pages/storage/StoragePoolSelector.tsx

```diff
@@ -74,8 +74,6 @@ const StoragePoolSelector: FC<Props> = ({
 
   return (
     <Select
-      id="storage-pool-selector"
-      name="pool"
       options={getStoragePoolOptions()}
```

Comment:
I was probably commenting a bit ambiguous before. I meant customizing those for the profile form use case should be avoided. We should keep an id and a name here, otherwise we end up with trouble for a11y.